components/AssetsList/BaseGovernanceForm.tsx

```diff
@@ -15,16 +15,29 @@ import {
 } from '@tools/sdk/units'
 import BigNumber from 'bignumber.js'
 import React, { useEffect, useState } from 'react'
+import { BaseGovernanceFormFieldsV3 } from './BaseGovernanceForm-data'
+import { BaseGovernanceFormV3 } from './BaseGovernanceFormV3'
 
-export interface BaseGovernanceFormFields {
+export interface BaseGovernanceFormFieldsV2 {
```

Comment:
Throughout the code, wherever data is being passed around, I annotate the interfaces with `_programVersion`. This lets us have strictly typed data structures and components/parsers can just discriminate on the `_programVersion` field. I think this is the way we should handle versioning in the codebase, rather than having a bunch of optional fields and hidden-from-compiler logic for when they are required.